idea-plugin/src/main/kotlin/arrow/meta/ide/dsl/utils/IdeUtils.kt

```diff
@@ -75,6 +73,9 @@ val KtElement.callElements: List<KtCallElement>
 val KtElement.typeReferences: List<KtTypeReference>
   get() = sequence(KtTypeReference::class.java)
 
+val KtTypeReference.argumentList: List<KtTypeReference>
```

Comment:
we should restrict further additions to this file if it is simply `sequence`. This is fine, for readers to understand how sequence work for the future we don't want to maintain those properties, as it may collide with user perspectives and naming issues, that may come up.